code/modules/power/power.dm

```diff
@@ -188,7 +188,7 @@
 // return a knot cable (O-X) if one is present in the turf
 // null if there's none
 /turf/proc/get_cable_node()
-	if(!istype(src, /turf/simulated/floor))
+	if(!istype(src, /turf))

```

Comment:
don't really think this check is even needed anymore.

It's a /turf proc, it can only be run on /turfs, if someone runs it on a non turf, they deserve it.
istype()ing src is stupid.